### DISCUSSION TITLE
Another suggestion

### DISCUSSION THREAD

**Discussion by mstick99** [lart2150/kmttg#20] [Ideas]
To the Remote tab section's Deleted tab add Export (to csv) option.

**Comment by lart2150** [comment]
I like this idea and it should be easy to add.

**Comment by lart2150** [comment]
Were you thinking something like this?  Can you think of any other fields you want pulled out into the csv?  I would likely remove the JSON field at the end.

[Bolt_deleted.csv](https://github.com/lart2150/kmttg/files/14233331/Bolt_deleted.csv)

**Reply by mstick99** [reply]
Shall I mark this discussion (and the others) as close?

**Comment by mstick99** [comment]
Looks good to me. One thing -- maybe include the episode title if available.

**Comment by mstick99** [comment]
Decided not to implement?

**Reply by lart2150** [reply]
it's in the repo and will be part of the next release. https://github.com/lart2150/kmttg/commit/26e1240826815c8c75cccde89fa37141f1b21c20

**Reply by mstick99** [reply]
outstanding!
